source/gtd/GirWrapper.d

```diff
@@ -585,56 +588,56 @@ class GirWrapper
 		}
 	}
 
-	private string getGirDirectory()
+	private string[] getGirDirectories()
 	{
-		version(Windows)
+		static string[] dirs;
+		if(dirs !is null)
 		{
-			import std.process : environment;
-
-			static string path;
+			return dirs;
+		}
+		else
+		{
+			dirs = [];
+		}
 
-			if (path !is null)
-				return path;
+		if ( commandlineGirPath )
+		{
+			dirs ~= commandlineGirPath;
+		}
 
+		version(Windows)
+		{
 			foreach (p; splitter(environment.get("PATH"), ';'))
 			{
 				string dllPath = buildNormalizedPath(p, "libgtk-3-0.dll");
 
-				if ( exists(dllPath) )
```

Comment:
The check to see if the path to de dll's exists should stay, or else you will create a lot of bogus paths.

Reply:
I think this was fixed by https://github.com/gtkd-developers/gir-to-d/pull/37/commits/df3895d13030d7a71dce5a97db37e2afcba0fd64